### PR TITLE
created 3by3 matrix fragment

### PR DESCRIPTION
**Pull request by alexisc99**
Should now take bundle arguments to create MxN matrices

### CODE REVIEW

File: app/src/main/java/ch/epfl/qedit/view/question/TableFragment.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+/**
+ * A simple {@link Fragment} subclass. Use the {@link TableFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class TableFragment extends Fragment {
+    private int sizeX;
+    private int sizeY;
+    private List<View> answers;
+
+    private Button buttonSubmit;
```
Comment (nbaehler) [inline app/src/main/java/ch/epfl/qedit/view/question/TableFragment.java]: We don't need this button.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -9,4 +9,8 @@
     <string name="role">"Display role"</string>
     <string name="buttonOpenQuiz">"Quiz test"</string>
     <string name="buttonLogin">"Connexion"</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
```
Comment (nbaehler) [inline app/src/main/res/values/strings.xml]: We probably can delete these lines.

File: app/src/main/java/ch/epfl/qedit/view/question/TableFragment.java (hunk 1 above)
Comment (alexisc99) [reply app/src/main/java/ch/epfl/qedit/view/question/TableFragment.java]: Yes, I forgot to remove it. Done now.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (alexisc99) [reply app/src/main/res/values/strings.xml]: Of course

**Pull request closed without merging**